src/voxelalgorithms.h

```diff
@@ -110,13 +110,15 @@ struct VoxelLineIterator
 	 */
 	v3s16 m_step_directions { 1, 1, 1 };
 	//! Position of the current node.
-	v3s16 m_current_node_pos;
+	v3s16 m_current_node_pos { 0, 0, 0 };
 	//! Index of the current node
 	s16 m_current_index = 0;
 	//! Position of the start node.
-	v3s16 m_start_node_pos;
+	v3s16 m_start_node_pos {0, 0, 0};
```

Comment:
FWIW, this is already ensured by the default constructor of the class.

Reply:
Indeed